pandas/tests/series/test_analytics.py

```diff
@@ -32,36 +32,64 @@ class TestSeriesAnalytics(TestData):
     @pytest.mark.parametrize("use_bottleneck", [True, False])
     @pytest.mark.parametrize("method", ["sum", "prod"])
     def test_empty(self, method, use_bottleneck):
+        if method == "sum":
```

Comment:
you should just pass in unit with method

``@pytest.mark.parametrize("method, unit", [("sum", 0.0), ("prod", 1.0)]))``